### ISSUE TITLE
Artillery / Fargate: Worker does not receive go signal and waits endlessly 

### ISSUE THREAD

**Issue by 70mm1** [artilleryio/artillery#3207]
When running Artillery with Distributed Testing and Fargate, everything starts as expected. The worker switches to last status `RUNNING` but then, nothing happens. The test is started from Jenkins, where it stops at `Waiting for Fargate...`. 
The go signal is not received and it is stuck in an endless loop of waits. 

I put the go.json file manually in the S3 bucket to test the rest and it worked as expected. The middle part does not currently. 
Any hints of what I am missing is very much appreciated. 

Version info: 
```
Artillery v2.0.15 
```

Running this command: 
```
run-fargate \
 --region *** \
--subnet-ids *** \
 --security-group-ids ***  \
--count 1 \
--cluster artilleryio-cluster \
--spot \
/load-test/load-test.yml
```

I expected to see this happen:
The Artillery worker launches the load test. 

Instead, this happened:
Instead it is stuck at `Waiting for green signal` and waiting for the go.json file.

**Comment by 70mm1** [comment]
Yes, I oversaw the 400 status code in the describeTasks debug logs. Root cause was that I tried to scope task permissions on the resource level which is not supported. :D So all good, error was in front of the computer.

**Comment by hassy** [comment]
ah okay, thanks for letting us know! We need to do better with detecting and reporting errors like that for sure

**Comment by 70mm1** [comment]
I would very like to support and contribute, is this possible in any way or a channel to orchestrate such efforts? @hassy
